internal/ingress/status/status.go

```diff
@@ -165,7 +165,12 @@ func NewStatusSyncer(config Config) Syncer {
 // ingress controller is currently running
 func (s *statusSync) runningAddresses() ([]string, error) {
 	if s.PublishStatusAddress != "" {
-		return []string{s.PublishStatusAddress}, nil
+		multipleAddrs := strings.Split(s.PublishStatusAddress, ",")
```

Comment:
it'd be nice to support lists like `ip1, ip2` where there's also space after comma. you can do regexp split here based on the pattern of `,\s*`

Reply:
ok, fixed it in 7e140f